src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -58,6 +59,7 @@ protected function configure()
                 new InputOption('show-controllers', null, InputOption::VALUE_NONE, 'Show assigned controllers in overview'),
                 new InputOption('format', null, InputOption::VALUE_REQUIRED, 'The output format (txt, xml, json, or md)', 'txt'),
                 new InputOption('raw', null, InputOption::VALUE_NONE, 'To output raw route(s)'),
+                new InputOption('sort', null, InputOption::VALUE_REQUIRED, 'The sorting field (priority, name, or path)', 'priority'),
```

Comment:
should this be declared as an enum? from the doc:
![image](https://user-images.githubusercontent.com/243674/86117211-e9fe2e00-bace-11ea-9738-083f47f706db.png)

Reply:
Thanks @nicolas-grekas, done in 7af952f99dfaefcee09ede201ce1a776d1d5f07d.
Should I change it also for the `format` option?

Reply:
@nicolas-grekas, at the moment the sorting function acts on a single field.
By accepting an array do you mean that using `$ console debug:router --sort path name`, internally it will sort first by name then by path?
If it's so, then I can try to implement it, evaluating some particular cases, like this: `--sort priority name path` should return the routes sorted only by priority.

Reply:
I fooled myself and you next I fear. I had the illusion that this would declare an enum, that's not the case. Sorry for the confusion.

Reply:
No problem :-) Resolving this one